LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/BasicsStation/BasicsStationConfigurationService.cs

```diff
@@ -74,9 +75,9 @@ public async Task<string> GetRouterConfigMessageAsync(StationEui stationEui, Can
                 var configJson = desiredProperties[RouterConfigPropertyName].ToString();
                 return LnsStationConfiguration.GetConfiguration(configJson);
             }
-            catch (ArgumentOutOfRangeException)
+            catch (ArgumentOutOfRangeException ex)
```

Comment:
Why not just do a contains check rather than using exception handling? This `catch` technically spans over much more than the desired properties access so it could misreport same errors during JSON reading of the configuration as twin missing the configuration.

Reply:
Done